src/app/space/app-launcher/services/app-launcher-gitprovider.service.spec.ts

```diff
@@ -51,10 +50,9 @@ describe('Service: AppLauncherGitproviderService', () => {
     controller = TestBed.get(HttpTestingController);
   });
 
-  it('should get GitHubDetails', (done: DoneFn) => {
-    service.getGitHubDetails().subscribe((val) => {
+  it('should get GitHubDetails', async () => {
```

Comment:
How does this `async` work without an `await`?
Before we notified done within the `subscribe`.